src/olympia/abuse/serializers.py

```diff
@@ -194,6 +197,13 @@ def get_addon(self, obj):
                 pass
         return output
 
+    def create(self, validated_data):
+        instance = super().create(validated_data)
+        if validated_data.get('reason') in AbuseReport.REPORTABLE_REASONS:
+            # call task to fire off cinder report
+            report_to_cinder.delay(instance.id)
```

Comment:
We need a waffle switch or setting to avoid enabling this on prod for the time being, right ? (Preferably here to avoid creating useless `CinderReport` instances that would not be sent yet)

Reply:
https://github.com/mozilla/addons-server/pull/21287/commits/cc7a2039ea8507975bc75330ccd5108278f0d7de